packages/nulp_elite/src/pages/connections/chat.js

```diff
@@ -127,7 +127,7 @@ const Chat = ({
   const [eneteredtextValue, setEnteredTextValue] = useState("");
   const charLimit = 700;
   const [prefilledMessage, setPrefilledMessage] = useState(
-    t("HELLO_CONNECT_MESSAGE")
+    "Hello! I’d like to connect with you."
```

Comment:
add lang const

Reply:
@paritshivani we haven't used this variable anywhere in code someone has declared it and not used